app/views/posts/index.html.erb

```diff
@@ -1 +1,26 @@
-<h1>This is lits posts<h1>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+  <head>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;700&display=swap" rel="stylesheet">
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+  </head>
+
+  <body>
+    <div class="root-container posts-comments-container">
+      <%= render 'shared/user_info', user: @user %>
+      <div class="posts-comments">
+        <% @posts.each_with_index do |post, post_no| %>
+          <div class="post-comments">
+            <%= render 'shared/post_info', post: post, post_no: post_no %>
+            <div class="comments">
+              <% post.comments.each do |comment| %>
+                <%= render 'shared/comment', comment: comment %>
+              <% end %>
+            </div>
+          </div>
+        <% end %>
+        <button class="btn">Pagination</button>
```

Comment:
- Can you use the` five_recent_comments` method to render the most recent comments as stated in the requirements? 
- Make Pagination bottom navigate to where it is supposed to and not manipulate the URL manually to move to the next page
![4](https://github.com/Paul-tes/Blog-App/assets/63972670/750af659-e780-4777-a75d-83579c9f8de3)
- Wireframe
![5](https://github.com/Paul-tes/Blog-App/assets/63972670/c66228e5-988f-4657-b873-d0518c44ba47)